dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/SystemController.java

```diff
@@ -237,7 +239,11 @@ public void getTaskJsonByUid( @PathVariable( "jobType" ) String jobType, @PathVa
         setNoStore( response );
         response.setContentType( CONTENT_TYPE_JSON );
 
-        renderService.toJson( response.getOutputStream(), notifications );
+        // Reverse order to become new->old sorted
```

Comment:
Maybe remove comment, quite clear what is happening. Or move to separate method with descriptive name.